index.html

```diff
@@ -23,16 +24,209 @@ <h1>Hey there. I'm Emmanuel <br>
                 I can help you build a product , feature or website Look 
                 through some of my work and experience! If you like what you see
                  and have a project you need coded, 
-                don’t hestiate to contact me.
+                don't hestiate to contact me.
             </p>
         </div>
         <div class="handle-icon">
-            <img class="svg" src="twitter.svg" alt="twitter logo">
-            <img class="svg" src="Linkedin-icon.svg" alt="LinkedIn logo">
-            <img class="svg" src="medium.svg" alt="Medium logo">
-            <img class="svg" src="github.svg" alt="Github logo">
-            <img class="svg" src="angellist.svg" alt="angellist logo">
+            <img class="svg" src="images/twitter.svg" alt="twitter logo">
+            <img class="svg" src="images/Linkedin-icon.svg" alt="LinkedIn logo">
+            <img class="svg" src="images/medium.svg" alt="Medium logo">
+            <img class="svg" src="images/github.svg" alt="Github logo">
+            <img class="svg" src="images/angellist.svg" alt="angellist logo">
         </div>
+        
     </section>
+
+    <div class="action">
+        <img src="images/Action.svg" alt="">
+    </div>
+    
+    <section class="recent-work">
+            <header>
+                <h2>My Recent <br> Works</h2>
+            </header>
+    
+            <div class="indicator">
+                <img class="work-icon" src="images/Indicator.svg" alt="">
+            </div>
+            
+    
+            <div class="cards">
+                <div class="card"></div>
+    
+                <div class="card-details">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+    
+                    <ul class="card-tag" >
+                        <li class="tags ruby">Ruby on rails</li>
+                        <li class="tags css">css</li>
+                        <li class="tags jav">JavaScript</li>
+                        <li class="tags htm">html</li>
+                    </ul>
+
+    
+                    <div class="btn">
+                        <button class="button" type="button">See Project</button>
+                    </div>
```

Comment:
- [ ] Kindly follow the button styling: Active, Hover, and Pressed (see design).
### Design
![figma](https://user-images.githubusercontent.com/81727988/176996634-00a583fd-5633-4e3c-a3c7-4bf5c029c89d.png)

Reply:
I have fix this error